typings/ember/ember.d.ts

```diff
@@ -21,7 +21,7 @@ declare module 'ember' {
      */
```

Comment:
Lets remove this file and add it to the list in `.gitignore`. Maybe we should just add the whole `typings/` directory.